app/src/main/res/layout/fragment_pledge_section_bonus_support.xml

```diff
@@ -129,6 +129,7 @@
             android:imeOptions="actionDone"
             android:importantForAutofill="no"
             android:inputType="numberDecimal"
+            android:digits="0123456789.,"
```

Comment:
This allows the user to enter both `.` and `,` characters, before it would only accept `.`

Reply:
Do we want to accept , though? I noticed it's disabled on web, plus it doesn't behave the way a EU person would expect it to behave. Might as well just disable it?

Reply:
QA note: must change device language too!